Bottom: 7272e42eaa5f911fa4e114b67fb12099a278a57d
Top:    3f4ba48a611f719563dc773a1c33fbd391708df0
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-14 16:48:24 -0700

Refresh of fstat

---

diff --git a/src/preload.c b/src/preload.c
index cc2162a..8f19af5 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -1040,6 +1040,7 @@ int __fxstat(int ver, int socket, struct stat *buf)
 {
 	int fd, ret;
 
+	init_preload();
 	if (fd_get(socket, &fd) == fd_rsocket) {
 		ret = real.fxstat(ver, socket, buf);
 		if (!ret)
